lib/src/main/java/com/what3words/androidwrapper/voice/VoiceApiListener.kt

```diff
@@ -6,6 +6,7 @@ import com.what3words.javawrapper.response.Suggestion
 /**
  * Implement this listener to receive the callbacks from VoiceApi
  */
+@Deprecated("This class is deprecated, please use new W3WApiVoiceDatasource instead")
```

Comment:
@phpduy, can I ask you to use ReplaceWith on deprecation, please? To help this migration, here are some links:

https://dev.to/mreichelt/the-hidden-kotlin-gem-you-didn-t-think-you-ll-love-deprecations-with-replacewith-3blo
https://stackoverflow.com/questions/52202810/how-does-deprecated-replacewith-work-for-kotlin-in-intellij

Reply:
I thought of using it before but I realized that ReplaceWith is only helpful when the old and new functions have similar signatures or when parameters from the old function can be directly mapped to the new one. In our case, we are introducing new APIs and classes and the way they work is different from the deprecated ones. In such cases, manual refactoring or other migration strategies might be necessary to handle the differences between the old and new code. Correct me if I'm wrong mate @mani-w3w

Reply:
@phpduy that makes sense mate 👍🏻